### PR TITLE
Update instructions to reflect that the bugfix is provided

### PR DESCRIPTION
**Pull request by cadichris**
I removed the "sneek peaks" content.
I thought links to the source code were enough, since we now have both the buggy code and the fixed code there.

I'm note sure we still need the section about the _Repository_ in the second markdown file.

### CODE REVIEW

**Review by mengdaming — APPROVED** [review]

**Review by aatwi — COMMENTED** [review]
For me, the code snippet will allow the users to view the fixes while reading and remaining on the instructions page.  
If we just reference the file, then they will have to switch to another page or go to the IDE. 

I believe that the docs folder on github should include as much information as possible for people to run it on their own. 
For us as facilitators, we can reference the Miro files to provide a summary of the story when running it in a short time session like the DDD.

**Comment by cadichris** [comment]
Thanks for the feedback @aatwi 

The .md now looks like this :

image

File: docs/2_Easy_Fix_Difficult_Test.md [hunk 1]
```diff
@@ -228,33 +212,22 @@ in the code and provided us with quick fixes!
...
-So let's revert!
-
-### 3. Write a test on Invoice and only then fix it
+## Now we need Unit Tests
 
-Let's add the test to 
-[Invoice](../java/src/main/java/com/murex/tbw/purchase/Invoice.java), reproduce the
-issue, and fix the code.
+### 1. On the Invoice
+Your mission is to add a test to
```
Comment (aatwi) [inline docs/2_Easy_Fix_Difficult_Test.md:218]: Small typo here :) for instead of to

Comment (aatwi) [reply docs/2_Easy_Fix_Difficult_Test.md:218]: 'to add a test for ...'

**Review by aatwi — COMMENTED** [review]
small typo change

**Review by mengdaming — APPROVED** [review]

**Comment by cadichris** [comment]
Thanks @aatwi. Typo is fixed.

**Pull request merged**